examples/photoCalTask.py

```diff
@@ -30,8 +30,8 @@
 import sys
 import lsst.afw.table              as afwTable
 import lsst.afw.image              as afwImage
-from lsst.meas.astrom import AstrometryTask
 from lsst.pipe.tasks.photoCal import PhotoCalTask
+import lsst.meas.astrom as measAstrom
```

Comment:
No big deal, but why not just import the one additional symbol from meas.astrom?